code/modules/cargo/orderconsole.dm

```diff
@@ -322,21 +346,6 @@
 				usr.investigate_log("called the supply shuttle.", INVESTIGATE_CARGO)
 				say("The supply shuttle has been called and will arrive in [SSshuttle.supply.timeLeft(600)] minutes.")
 				SSshuttle.moveShuttle(cargo_shuttle, docking_home, TRUE)
-			if(!length(cart_list))
```

Comment:
Now there'll have a paper even if nothing was purchased, was this intentional?

Reply:
No, fixed the issue. No paper will be generated when nothing is in the shopping_list.